doc/bgm.md

```diff
@@ -1,96 +1,109 @@
 # Looping Background Music
 
-With the `Bgm` class you can manage looping of background music tracks with regards to application (or game) lifecycle state changes.
+With the `Bgm` class you can manage looping of background music tracks with regards to application
+(or game) lifecycle state changes.
 
-When the application is paused, terminated, or sent to background, `Bgm` will automatically pause the currently playing music track. Similarly, when the application is resumed, `Bgm` will resume the background music. Manual pause and resume is also supported.
+When the application is paused, terminated, or sent to background, `Bgm` will automatically pause
```

Comment:
when is an application paused? Is that a standard mobile term? I think it's not super clear but might be just me.

Reply:
Removing that part, not something I wrote.